core/src/test/scala/unit/kafka/raft/KafkaNetworkChannelTest.scala

```diff
@@ -139,6 +145,25 @@ class KafkaNetworkChannelTest {
     }
   }
 
+  @Test
+  def testNonRoutableAddressUpdateRequest(): Unit = {
+    val destinationId = 2
+    assertThrows(classOf[IllegalArgumentException],
+      () => new InetAddressSpec(new InetSocketAddress("0.0.0.0", 0)))
+
+    // Update channel with a valid endpoint
```

Comment:
Not sure there's much value in the rest of this test. Seems effectively the same as `testSendAndReceiveOutboundRequest`.

Reply:
Fair enough. Removed.